airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -1359,6 +1359,128 @@ def execute(self, context):
         super().execute(context)
 
 
+class DataprocSubmitSparkRJobOperator(DataprocJobBaseOperator):
```

Comment:
Do we need custom operator for R jobs? Can't we use the `DataprocSubmitJobOperator`?

Reply:
The `DataprocJobBaseOperator` as well as other XJobOperators will be deleted in future:
```
        # TODO: Remove one day
        warnings.warn(
            "The `{cls}` operator is deprecated, please use `DataprocSubmitJobOperator` instead. You can use"
            " `generate_job` method of `{cls}` to generate dictionary representing your job"
            " and use it with the new operator.".format(cls=type(self).__name__),
            DeprecationWarning,
            stacklevel=1
        )
```

Reply:
Is there an example of how to use the generic DataprocJobBaseOperator for all the jobs yet, @turbaszek?

Reply:
https://github.com/apache/airflow/blob/master/airflow/providers/google/cloud/example_dags/example_dataproc.py#L141-L166

Reply:
oh that is way easier, I'll create a new PR